core/build.gradle

```diff
@@ -57,4 +57,9 @@ dependencies {
     compile "io.reactivex:rxandroid:$rxAndroidVersion"
 
     compile "com.github.AntennaPod:AntennaPod-AudioPlayer:$audioPlayerVersion"
+
+    // Add casting features
+    compile "com.google.android.libraries.cast.companionlibrary:ccl:$castCompanionLibVer"
+    //compile "com.android.support:mediarouter-v7:$supportVersion"
+    //compile "com.google.android.gms:play-services:$playServicesVersion"
```

Comment:
In principle, since the commented ones are dependencies of cast companion library, we don't seem to need to compile them.